core-primitives/substrate-sgx/sp-io/src/lib.rs

```diff
@@ -941,4 +942,12 @@ mod tests {
 
 		assert_eq!(ext.get(CODE).unwrap(), &code);
 	}
+
+	#[test]
+	#[should_panic(
+		expected = "`set` cannot be called outside of an Externalities-provided environment."
+	)]
+	fn storage_set_without_externalities_panics() {
+		storage::set(b"hello", b"world");
+	}
```

Comment:
What I've read, is that using 
```
panic::catch_unwind(|| {
		...
	}
```
is preferred over `#[should_panic(`, because it gives you better control over which part exactly panics, and you can as many assertions as you like to ensure it's the correct panic.

Reply:
I see that makes sense and I did not know that, thanks! However, I think in this very simple case, it should be enough to just add the `#[should_panic]` flag, do you disagree?

Reply:
We can leave it as is here - just wanted to make sure we know that there is an alternative ☺️

Reply:
Happy you agree. :)